src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -2538,7 +2538,10 @@ export class FeatureToggleService {
                         environment,
                     ));
                 if (!canAddStrategies) {
-                    throw new PermissionError(CREATE_FEATURE_STRATEGY);
+                    throw new PermissionError(
+                        CREATE_FEATURE_STRATEGY,
+                        environment,
```

Comment:
This is a little detail. `CREATE_FEATURE_STRATEGY` is an environment permission. Including the environment here provides a better error message.